docs/reference/bus-topics.md

```
@@ -435,24 +435,24 @@ These sites pass a non-literal topic that the static scan couldn't resolve to a
 | `lib/plugins/world-state-engine.ts:418` | publish | `domainTopic` |
 | `lib/plugins/world-state-engine.ts:695` | publish | `replyTopic` |
 | `lib/plugins/world-state-engine.ts:713` | publish | `topic` |
-| `src/api/a2a-server.ts:130` | publish | `{
+| `src/api/a2a-server.ts:133` | publish | `{
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)